app/Resources/views/lift/_leaderboard.html.twig

```diff
@@ -11,7 +11,7 @@
         <tr>
             <td class="text-center">{{ leader.username }}</td>
             <td class="text-center">{{ leader.weight }}lbs</td>
-            <td class="text-center">{{ leader.in_cats|number_format(0) }} fat cats</td>
+            <td class="text-center">{{ leader.in_cats }} fat cats</td>
```

Comment:
I'm not sure we have to merge it because it's an edge case and as the result, we will mismatch the video. What do you think about just adding a note instead, which explains problems with huge numbers?